e2e/integration/list-requests.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('check the lists of requests and donators are there and not empty', () => {
```

Comment:
while we're at it, separate into 2 tests with better descriptions that it('passes)?

Reply:
Ah yes, that description was auto-generated and I didn't spot it! I'll separate things out better (and describe them better!) in further commits.